src/standalone/Snow/Snow.jl

```diff
@@ -42,6 +42,81 @@ and is used as a bulk snow model.
 """
 abstract type AbstractSnowModel{FT} <: ClimaLand.AbstractExpModel{FT} end
 
+"""
+    AbstractDensityModel{FT}
+Defines the model type for density and depth parameterizations
+for use within an `AbstractSnowModel` type. Current examples include the
+`ConstantDensityModel` and the `Anderson1976` models.
+"""
+abstract type AbstractDensityModel{FT <: AbstractFloat} end
+
+"""
+    ConstantDensityModel{FT <: AbstractFloat} <: AbstractDensityModel{FT}
+Establishes the density parameterization where snow density
+is always treated as a constant (type FT).
+"""
+struct ConstantDensityModel{FT} <: AbstractDensityModel{FT}
+    ρ_snow::FT
+end
+
+"""
+    ConstantDensityModel{FT}(ρ::FT)
```

Comment:
We may not need this outer constructor....the default would be:

ConstantDensityModel(rho) and it would use the type of rho as FT

is this ok to use? Or is your intent to make sure the type of rho matches the FT set in the sim script?

Reply:
You are correct, no outer constructor is needed